### DISCUSSION TITLE
Course content review display

### DISCUSSION THREAD

**Discussion by manjirisulekavadi** [Sunbird-Ed/Community#576] [Enhancement request]
Course review can have comments at two levels 
1) At the course level overall
2) Or at the content level (even if it is reviewed and published)
Is it possible to tag the review comments in a collection at the resource level and display the comments at overall level and at the content level separately as applicable for the creator. 
@Krishnaj20

**Comment by Krishnaj20** [comment]
@manjirisulekavadi - currently review comments are at the collection level or content level depending on whether the reviewer is reviewing a collection or content. We can help you if you would like to enhance this experience and contribute back.   

@ashokreddy1208 - Pls add more details.

**Comment by ashokreddy1208** [comment]
@manjirisulekavadi  Review can add comments to the individual contents while publishing/rejecting contents.  reviewer can add only approved/published contents to the collection.

Right now there is no option for reviewer to comment on contents if he/she is reviewing collection.

cc @Krishnaj20

**Reply by manjirisulekavadi** [reply]
Thanks @ashokreddy1208 will discuss the priority of a use case for content level review comments in the collection - though the content (resource) is published.     Discussion was about in a case when the published content is partially applicable for the course and hence needs to be split.
